tukaan/_tcl.py

```diff
@@ -30,17 +32,13 @@ def __init__(self, callback: Callable[..., Any], converters=(), args=(), kwargs=
         Tcl._interp.createcommand(name, self.__call__)
 
     def __call__(self, *tcl_args) -> Any:
-        if self._converters and tcl_args:
-            result = []
-            for index, value in enumerate(tcl_args):
-                with contextlib.suppress(KeyError):
-                    value = Tcl.from_(self._converters[index], value)
-                result.append(value)
-
-            tcl_args = tuple(result)
-
         try:
-            return self._callback(*tcl_args, *self._args, **self._kwargs)
+            if inspect.iscoroutinefunction(self._callback):
```

Comment:
This is bad here. `TclCallback.__call__` is already too slow. Maybe an `AsyncTclCallback` class?